sql-ballerina/Module.md

```diff
@@ -0,0 +1,371 @@
+
+
+### Client
+
+The database client should be created using any of the above-listed database modules and once it is created, the 
+operations and functionality explained below can be used. 
+
+#### Connection pool handling
+
+All database modules share the same connection pooling concept and there are 3 possible scenarios for 
+connection pool handling.  For its properties and possible values, see the `sql:ConnectionPool`.  
+
+1. Global shareable default connection pool
+
+    If you do not provide the `poolOptions` field when creating the database client, a globally-shareable pool will be 
+    created for your database unless a connection pool matching with the properties you provided already exists. 
+    The JDBC module example below shows how the global connection pool is used. 
+
+    ```ballerina
+    jdbc:Client|sql:Error dbClient = 
+                               new ("jdbc:mysql://localhost:3306/testdb", 
+                                "root", "root");
+    ```
+
+2. Client owned, unsharable connection pool
+
+    If you define the `connectionPool` field inline when creating the database client with the `sql:ConnectionPool` type, 
+    an unsharable connection pool will be created. The JDBC module example below shows how the global 
+    connection pool is used.
+
+    ```ballerina
+    jdbc:Client|sql:Error dbClient = 
+                               new (url = "jdbc:mysql://localhost:3306/testdb", 
+                               connectionPool = { maxOpenConnections: 5 });
+    ```
+
+3. Local, shareable connection pool
+
+    If you create a record of type `sql:ConnectionPool` and reuse that in the configuration of multiple clients, 
+    for each  set of clients that connects to the same database instance with the same set of properties, a shared 
+    connection pool will be created. The JDBC module example below shows how the global connection pool is used.
+
+    ```ballerina
+    sql:ConnectionPool connPool = {maxOpenConnections: 5};
+    
+    jdbc:Client|sql:Error dbClient1 =       
+                               new (url = "jdbc:mysql://localhost:3306/testdb",
+                               connectionPool = connPool);
+    jdbc:Client|sql:Error dbClient2 = 
+                               new (url = "jdbc:mysql://localhost:3306/testdb",
+                               connectionPool = connPool);
+    jdbc:Client|sql:Error dbClient3 = 
+                               new (url = "jdbc:mysql://localhost:3306/testdb",
+                               connectionPool = connPool);
+    ```
+    
+#### Closing the client
+
+Once all the database operations are performed, you can close the database client you have created by invoking the `close()`
+operation. This will close the corresponding connection pool if it is not shared by any other database clients. 
+
+```ballerina
+error? e = dbClient.close();
+if (e is error){
+    io:println("Error occured:", e);
+}
```

Comment:
remove these lines

Reply:
removed in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/d3df4347c964f2a634909920ca548fefdfc1139c